src/core-func-ext-async/Func.Extensions.Async.Tests/AsyncFuncTests.2/AsyncFuncTests.00.cs

```diff
@@ -0,0 +1,53 @@
+#nullable enable
```

Comment:
Shell we remove nullable enable directives from all files in .NET 6 project?

Reply:
Done.